bibliography/index.qmd

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Bibliography"
-title-block-banner: ../images/heros/biblio_hero.png
```

Comment:
Probably should remove the old one from the repo

Reply:
Good point, thanks!